mne/viz/brain/tests/test_brain.py

```diff
@@ -0,0 +1,68 @@
+import mne
```

Comment:
please check order of imports and make sure everyfile has a list of authors and a license